types/framework/entities/item.d.ts

```diff
@@ -70,19 +76,21 @@ declare class Item<D extends Item.Data = Item.Data<any>> extends Entity<D> {
    * @param effects - The raw array of active effect objects
    * @returns The prepared active effects collection
    */
-  _prepareActiveEffects(effects: ActiveEffect[]): Collection<ActiveEffect>;
+  protected _prepareActiveEffects(effects: ActiveEffect[]): Collection<ActiveEffect>;
```

Comment:
Similar here, this get's an array of data of active effects is parameter. fixed in #216